renovate.json

```diff
@@ -3,7 +3,7 @@
 	"config:base"
 	],
 	"baseBranches": ["main", "3.x"],
-	"useBaseBranchConfig": "merge",
+	"useBaseBranchConfig": merge,
```

Comment:
Looks more correct with quotes. 😕

Reply:
yeah, I also thought so...but as recorded by @emmileaf in the release manager notes, renovateBot isn't working for 3.x branch. So looking at the config again, I thought maybe this could be the reason? not able to figure it out. Here's the documentation for it -> https://docs.renovatebot.com/configuration-options/#usebasebranchconfig